resources/assets/less/bem/beatmap-discussions-header-top.less

```diff
@@ -56,6 +56,30 @@
     }
   }
 
+  &__filter-group {
+    min-width: 0;
+    flex: 1 1 auto;
+
+    & + & {
+      margin-top: 10px;
+    }
+
+    @media @desktop {
+      align-self: flex-end;
+
+      & + & {
+        margin-top: 0;
+      }
+
+      &--stats {
+        flex: none;
+        display: flex;
+        flex-direction: column;
```

Comment:
something here seems to cause incorrect alignment

![](https://s.myconan.net/2018-05/firefox_2018-05-07_15-56-56.png)